src/test/java/pages/ApplePage.java

```diff
@@ -0,0 +1,59 @@
+package pages;
+
+import elements.Tag;
+import elements.Text;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+import properties.ConfigStorage;
+import util.WaitUtils;
+
+public class ApplePage extends BasePage {
+
+    private static final String APPLE_FILTER_ENDPOINT = "mobile?mfr%5B0%5D=apple";
+    private final String CATALOGUE_URL = ConfigStorage.getCatalogueUrl();
```

Comment:
CATALOGUE_URL is used for almost every page, can we move it to BasePage?

Reply:
moved to BasePage